accelerometer/Accelerometer.https.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Accelerometer Test</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="https://www.w3.org/TR/accelerometer/">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/generic-sensor/generic-sensor-tests.js"></script>
+<div id="log"></div>
+<iframe src="support-iframe.html" id="frame" style="display:none" sandbox="allow-scripts">
+</iframe>
+<script>
+
+function verify_sensor_reading(sensor, should_be_bull) {
+  if (should_be_bull) {
+    return sensor.x == null &&
+           sensor.y == null &&
+           sensor.z == null &&
+           sensor.timestamp == null;
+  }
+  return sensor.x != null &&
+         sensor.y != null &&
+         sensor.z != null &&
+         sensor.timestamp != null;
+}
+
+function verify_both_sensor_reading(sensor1, sensor2) {
+  return sensor1.x == sensor2.x ||
+         sensor1.y == sensor2.y ||
+         sensor1.z == sensor2.z ||
+         sensor1.timestamp == sensor2.timestamp;
+}
+
+async_test(t => {
+  let sensor1 = new Accelerometer({includeGravity: false});
```

Comment:
How about testing that sensor can be successfully constructed with `includeGravity` option?